pkg/collectsub/server/db/simpledb/simpledb.go

```diff
@@ -52,15 +53,17 @@ func (s *simpleDb) containsEntry(e *pb.CollectEntry) bool {
 func (s *simpleDb) AddCollectEntries(ctx context.Context, entries []*pb.CollectEntry) error {
```

Comment:
This has to be updated to match the interface: `GetCollectEntries(context.Context, []*pb.CollectEntryFilter, int64) ([]*pb.CollectEntry, error)`